ta/Makefile

```diff
@@ -5,6 +5,8 @@ ifeq ($O,)
 $(error output path should be specified when calling this makefile)
 endif
 
+LDFLAGS=
```

Comment:
With @jbech-linaro comments incorporated, this is also:
`Reviewed-by: Jerome Forissier <jerome.forissier@linaro.org>`